admin-ui/app/components/ThemeDropdown/ThemeDropdown.tsx

```
@@ -0,0 +1,116 @@
+    )
+
+    const menuStyle = useMemo(() => {
+      const style: React.CSSProperties = {}
+      if (minWidth) style.minWidth = typeof minWidth === 'number' ? `${minWidth}px` : minWidth
+      if (maxWidth) style.maxWidth = typeof maxWidth === 'number' ? `${maxWidth}px` : maxWidth
+      return style
+    }, [minWidth, maxWidth])
+
+    return (
+      <div className={`${classes.dropdownWrapper} ${className || ''}`} ref={dropdownRef}>
+        <div onClick={() => setIsOpen(!isOpen)} style={{ cursor: 'pointer' }}>
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)